Gaming Anthems/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": {
+    "name": "TechLewi",
+    "id": "590575157718941708"
+  },
+  "service": "GamingAnthems",

```

Comment:
Change the name of this to `Gaming Anthems`, the directory name of the presence must match.